apps/modernization-ui/src/styles/_uswds.scss

```diff
@@ -49,4 +50,18 @@ table.usa-table {
 // Default button styling
 button.usa-button {
     font-size: 1rem;
+}
+
+.search-result-item-label {
```

Comment:
issue:  This class doesn't belong here, it's not overriding a uswds style

Reply:
Okay